lms/djangoapps/onboarding/migrations/0024_re_populate_organization_metric_prompt.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from django.db import migrations, models
+from lms.djangoapps.onboarding.helpers import its_been_year, its_been_year_month, \
+    its_been_year_three_month, its_been_year_six_month
+from mailchimp_pipeline.signals.handlers import sync_metric_update_prompt_with_mail_chimp
+
+"""
+    NOTE: This migration is duplicate of '0020_populate_organization_metric_prompt'
+    Why we need to duplicate?
+    Ref: https://philanthropyu.atlassian.net/browse/LP-1311
+    While deploying LP-1222 on prod we forget to update `List fields and *|MERGE|* tags` in mailchimp's
+    PhilU Learners (From EdX API) list so now we have added the required fields in the list.
+
+"""
+
+
+def get_latest_metric(org):
+    """
+    :param org:
+    :return: latest organization metrics submitted by user
+    """
+    return org.organization_metrics.order_by('-submission_date').first()
+
+
+def get_latest_submission_date(org):
+    """
+    :param org:
+    :return: latest date of latest metric submission
+    """
+    return get_latest_metric(org).submission_date
+
+
+def get_responsible_user(org):
+    """
+    :param org:
+    :return: return admin of organization if exists otherwise the person with latest
+             metric submission is responsible user
+    """
+    return org.admin if org.admin else get_latest_metric(org).user
+
+
+def metric_exists(org):
+    """
+    :param org:
+    :return: True if some metrics exists against this organization, otherwise False
+    """
+    return bool(org.organization_metrics.all())
```

Comment:
use [exists()](https://docs.djangoproject.com/en/1.8/ref/models/querysets/#django.db.models.query.QuerySet.exists) instead of doing this.